healthcareai/trained_models/trained_supervised_model.py

```diff
@@ -67,6 +69,7 @@ def __init__(self,
         self.test_set_class_labels = test_set_class_labels
         self.test_set_actual = test_set_actual
         self._metric_by_name = metric_by_name
+        self._train_time = training_time
```

Comment:
I don't see the harm in making this a public class variable attribute rather than a private. Go ahead and change all references from `_train_time` to `train_time`